jsk_arc2017_baxter/euslisp/lib/pick-interface.l

```diff
@@ -240,10 +241,12 @@
         (send *ri* :angle-vector-raw (send *baxter* :angle-vector) 3000
               (send *ri* :get-arm-controller arm :gripper nil) 0)
         (send *ri* :wait-interpolation))
-      (if (eq graspingp :ik-fail) (return-from :pick-object nil))
-      (setq graspingp (send *ri* :graspingp arm grasp-style-))
-      (ros::ros-info "[main] arm: ~a graspingp: ~a" arm graspingp)
-      graspingp))
+      (if (eq pick-res :ik-fail) (return-from :pick-object nil))
```

Comment:
I think `graspingp` is fine if it has `:ik-fail` as value because `pick-result` means nothing. Or you can return two values `graspingp` and `ik-failp`.